README.md

```diff
@@ -21,7 +34,11 @@ cd fonttools ;
 python setup.py install ;
 ```
 
-Full build and installation instructions are in [Doc/install.txt](https://github.com/behdad/fonttools/blob/master/Doc/install.txt) 
+This will install all the modules in the right places. 
+On Unix it also installs the command line tools. 
```

Comment:
I removed the "Windows" folder as it was outdated (cf. https://github.com/behdad/fonttools/pull/423). We now rely on setuptools' `console_scripts` feature to create the platform-specific executable scripts. This is done automatically by the setup.py script upon install, so there's no need to differentiate Unix vs Windows in the installation instructions.

@davelab6  Could you please remove lines 40-41, and change line 37-38 so that it simply says something like:

```diff
-This will install all the modules in the right places. 
-On Unix it also installs the command line tools. 
+This will install all the modules and command line tools in the right places.
```